src/coffee/import.coffee

```diff
@@ -38,26 +39,34 @@ class Import
       @sync = new ProductSync
       @repeater = new Repeater attempts: 3
 
-    # TODO: move initialisation somewhere else
-    options.types = new Types()
-    options.customerGroups = new CustomerGroups()
-    options.categories = new Categories()
-    options.taxes = new Taxes()
-    options.channels = new Channels()
-
-    @validator = new Validator(options)
-    @map = new Mapping(options)
+    options.importFormat = options.importFormat || "csv"
+    options.csvDelimiter = options.csvDelimiter || ","
+    options.encoding = options.encoding || "utf-8"
+    @dryRun = false
+    @updatesOnly = false
     @publishProducts = false
     @continueOnProblems = options.continueOnProblems
     @allowRemovalOfVariants = false
-    @updatesOnly = false
-    @dryRun = false
     @blackListedCustomAttributesForUpdate = []
     @customAttributeNameToMatch = undefined
     @matchBy = CONS.HEADER_ID
     @options = options
+#    @initializeObjects()
```

Comment:
Initialization is now done when doing import - alias here:
https://github.com/sphereio/sphere-node-product-csv-sync/pull/137/files/9369cd729e709c08947c0fcd48fdb46f9bc1a363#diff-7ed9e504a5a8e3612516e56fc3dfd602R86